build.gradle

```diff
@@ -463,12 +463,10 @@ project(':iceberg-mr') {
     testCompile("org.apache.calcite:calcite-core")
     testCompile("com.esotericsoftware:kryo-shaded:4.0.2")
     testCompile("com.fasterxml.jackson.core:jackson-annotations:2.6.5")
-    testCompile("com.klarna:hiverunner:5.2.1") {
-      exclude group: 'javax.jms', module: 'jms'
+    testCompile("org.apache.hive:hive-service") {
       exclude group: 'org.apache.hive', module: 'hive-exec'
-      exclude group: 'org.codehaus.jettison', module: 'jettison'
-      exclude group: 'org.apache.calcite.avatica'
     }
+    testCompile("org.apache.tez:tez-dag:0.8.4")
```

Comment:
Is the tez dependency required? I don't see any imports explicitly referencing tez.

Reply:
You're right, thanks. It's not required for Hive2, indeed. I've removed the dependency.
On the other hand, it is needed for Hive3, where `HiveServer2.start()` initializes a `TezSessionPoolManager`, which depends on some Tez code.